lib/create.js

```diff
@@ -40,10 +40,8 @@ const create = async function create(functionName, functionPath,
   const funcConf = {
     ...YMLUtil.getFuncConf(binarisConf, templateName),
     runtime,
+    ...config,
```

Comment:
I think this means someone could technically overwrite their runtime with config.

Reply:
yes, you are correct (this is good for concise testing code)